src/repository/postDAO.js

```diff
@@ -0,0 +1,27 @@
+const uuid = require("uuid");
+const { PutCommand } = require("@aws-sdk/lib-dynamodb");
+
+const tableName = "";
+
+async function createPost(username, text, score){
+    if (score < 0 || !text){
+        return 1;
```

Comment:
returning numbers can be vague and hard to follow if you don't know what the developer who wrote it was thinking. You can maybe throw an error here or return an object such as {message: "Invalid score or text inputted"};

Reply:
Good point. I will look into return objects with messages.